pyaedt/application/Analysis3DLayout.py

```diff
@@ -236,40 +236,55 @@ def export_mesh_stats(self, setup_name, variation_string="", mesh_path=None):
         return mesh_path
 
     @pyaedt_function_handler()
-    def get_all_return_loss_list(self, excitation_names=[], excitation_name_prefix=""):
-        """Retrieve a list of all return losses for a list of excitations.
+    def get_all_return_loss_list(
+        self, excitation_names=None, excitation_name_prefix="", math_formula="", net_list=None
+    ):
+        """Retrieve a list of all return losses for a list of exctitations.
 
         Parameters
         ----------
         excitation_names : list, optional
-            List of excitations. The default is ``[]``, in which case
+            List of excitations. The default is ``None``, in which case
             the return losses for all excitations are to be provided.
-            For example, ``["1", "2"]``.
+            For example ``["1", "2"]``.
         excitation_name_prefix : string, optional
-             Prefix to add to the excitation names. The default is ``""``.
+             Prefix to add to the excitation names. The default is ``""``,
+        math_formula : str, optional
+            One of the available AEDT mathematical formulas to apply. For example, ``abs, dB``.
+        net_list : list, optional
+            List of nets to filter the output. If None, all parameters will be returned.
 
         Returns
         -------
-        list
+        list of str
             List of strings representing the return losses of the excitations.
-            For example, ``["S(1, 1)", "S(2, 2)"]``.
+            For example ``["S(1, 1)", S(2, 2)]``.
```

Comment:
```suggestion
            For example, ``["S(1, 1)", S(2, 2)]``.
```
The comma should be there to separate the introductory clause from the code entity.